org.telegram.desktop.yml

```diff
@@ -346,26 +326,34 @@ modules:
           versions:
             <: 2.73.0
 
-  - name: mozjpeg
```

Comment:
All official builds must use mozjpeg

Reply:
What really should become redundant with 22.08 are lcms2 and protobuf

Reply:
ok.
also as you said, lcms2 is already there, but protobuf isn't.
or at least building without it telegram-desktop fails to find it.

Reply:
:thinking: 
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/release/22.08/elements/components/protobuf.bst
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/release/22.08/elements/components/protobuf-base.bst
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/release/22.08/elements/include/protobuf-source.yml

Reply:
True, strange.
I will upload without it, maybe it only fails in my computer.
If it fails, I will commit it back in.

Reply:
I indeed can't find it in sdk through. Maybe they have the rules to build it, but don't add it to some root rule...

Reply:
I thought protobuf is necessary to build fontconfig (or freetype?) nowadays

Reply:
It seems they have it only for fwupd :cry: 
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/release/22.08/elements/components/fwupd.bst#L31

Reply:
It really [fails](https://buildbot.flathub.org/#/builders/6/builds/17339) without protobuf.

Reply:
yeah, apparently we still need to build it